pkgs/top-level/python-packages.nix

```diff
@@ -32038,6 +32038,26 @@ EOF
     };
   };
 
+  typed-ast = buildPythonPackage rec {
```

Comment:
Might be worth sending a mail to the mailing list! I think I missed the discussion :)